app/views/passenger_mailer/notify_pending.haml

```diff
@@ -0,0 +1,9 @@
+Dear #{@passenger.name},
+
+Your account with the Special Transportation Service has been created but is still pending. You can receive rides until #{(Time.now+(60*60*24*3)).strftime("%m/%d/%Y")}.
```

Comment:
We want three business days from now. Also there are helpers (sprinkled all over the code for example) that will get you the day you want without you having to do any math.